harbors/tests/test_harbors_graphql.py

```diff
@@ -1,31 +1,120 @@
+                  description
+                }
+                numberOfPlaces
+                maximumWidth
+                maximumLength
+                maximumDepth
+              }
             }
+          }
         }
+      }
     """
-    executed = old_schema_api_client.execute(query)
-    assert executed["data"]["boatTypes"] == [{"name": boat_type.name}]
-
-
-def test_get_harbors(harbor, old_schema_api_client):
-    query = """
-        {
-            harbors {
-                edges {
-                    node {
-                        properties {
-                            name
-                            zipCode
-                        }
-                    }
+
+
+def test_get_boat_type(boat_type, old_schema_api_client):
+    harbor = HarborFactory()
+
+    executed = old_schema_api_client.execute(CUSTOMER_UI_BOATS_BERTH_TYPES_QUERY)
+
+    harbor_node = executed["data"]["harbors"]["edges"][0]["node"]
+    assert harbor_node["id"] == to_global_id(HarborType._meta.name, harbor.id)
+    assert harbor_node["properties"]["name"] == harbor.name
+    assert harbor_node["properties"]["zipCode"] == harbor.zip_code
+
+    assert executed["data"]["boatTypes"] == [
+        {"id": str(boat_type.id), "name": boat_type.name}
+    ]
+
+
+CUSTOMER_UI_WS_AREAS_QUERY = """
+      query WinterAreasQuery {
+        winterStorageAreas {
+          edges {
+            node {
+              id
+              geometry {
+                coordinates
+              }
+              properties {
+                name
+                streetAddress
+                zipCode
+                imageFile
+                numberOfMarkedPlaces
+                maximumWidth: maxWidth
+                maximumLength: maxLength
+                numberOfSectionSpaces
+                servicemapId
+                maxLengthOfSectionSpaces
+                numberOfUnmarkedSpaces
+                electricity
+                water
+                gate
+                repairArea
+                summerStorageForDockingEquipment
+                summerStorageForTrailers
+                summerStorageForBoats
+                municipality
+                wwwUrl
+                availabilityLevel {
+                  id
+                  title
+                  description
                 }
+              }
             }
+          }
         }
+      }
     """
-    executed = old_schema_api_client.execute(query)
-    assert executed["data"]["harbors"]["edges"] == [
-        {"node": {"properties": {"name": harbor.name, "zipCode": harbor.zip_code}}}
-    ]
+
+
+def test_get_ws_areas(boat_type, old_schema_api_client):
+    area = WinterStorageAreaFactory()
+
+    executed = old_schema_api_client.execute(CUSTOMER_UI_WS_AREAS_QUERY)
+
+    area_node = executed["data"]["winterStorageAreas"]["edges"][0]["node"]
+    assert area_node["id"] == to_global_id(WinterStorageAreaType._meta.name, area.id)
+    assert area_node["properties"]["name"] == area.name
```

Comment:
Since we're querying for all the fields, would it make sense to assert that all of them are correct?

You can probably reuse some of the assertions from [resources/tests/test_resources_queries.py#L301](https://github.com/City-of-Helsinki/berth-reservations/blob/develop/resources/tests/test_resources_queries.py#L301)

Reply:
This PR was a first step that ensures that the query doesn't blow up. We can continue improving these tests when we have time.

Reply:
Alright! Let's keep it in mind then, but good progress already. :)